nixos/modules/services/continuous-integration/github-runner/service.nix

```diff
@@ -153,18 +157,21 @@ with lib;
             ln -s '${newConfigPath}' "${currentConfigPath}"
           fi
         '';
-        setupRuntimeDir = writeScript "setup-runtime-dirs" ''
+        setupWorkDir = writeScript "setup-work-dirs" ''
+          # Cleanup previous service
+          rm -rf "$WORK_DIRECTORY/*"
+
           # Link _diag dir
-          ln -s "$LOGS_DIRECTORY" "$RUNTIME_DIRECTORY/_diag"
+          ln -sf "$LOGS_DIRECTORY" "$WORK_DIRECTORY/_diag"
```

Comment:
Why is the `--force` required now? The link should never exist if the directory starts out empty.

Reply:
good catch!